polyply/tests/test_build_file_parser.py

```diff
@@ -0,0 +1,166 @@
+# Copyright 2020 University of Groningen
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Test that build files are properly read.
+"""
+import textwrap
+import pytest
+import numpy as np
+import vermouth.forcefield
+import vermouth.molecule
+import polyply
+from polyply.src.meta_molecule import Monomer, MetaMolecule
+from polyply.src.topology import Topology
+
+@pytest.mark.parametrize('tokens, expected', (
+   # for cylinder
+   (["PEO", "63", "154", "in", "5", "5", "5", "5", "5"],
```

Comment:
Also, don't use all 5 so you can be sure where the number ends up
```suggestion
   ("PEO 63 154 in 5 5 5 5 5".split(), 
```

Reply:
I prefer the explicit list over the string split